src/Core/Block.cpp

```diff
@@ -631,7 +631,7 @@ Names Block::getNames() const
     res.reserve(columns());
 
     for (const auto & elem : data)
-        res.push_back(elem.name);
+        res.push_back(std::move(elem.name));
```

Comment:
This looks wrong.
As the method is const, it should not destroy the Block's elements.